<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window_UpdateAccount</class>
 <widget class="QDialog" name="Window_UpdateAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 211, 148);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="1">
      <widget class="QLabel" name="label_ExpireDate">
       <property name="font">
        <font>
         <family>Microsoft Sans Serif</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 67, 99);</string>
       </property>
       <property name="text">
        <string>Expire Date</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="lineEdit_Username">
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(106, 101, 255);
color: rgb(216, 216, 216);</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string> Username</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_Username">
       <property name="font">
        <font>
         <family>Microsoft Sans Serif</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 67, 99);</string>
       </property>
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="lineEdit_Name">
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 121, 87);
color: rgb(216, 216, 216);</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string> Name       </string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_DateAdded">
       <property name="font">
        <font>
         <family>Microsoft Sans Serif</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 67, 99);</string>
       </property>
       <property name="text">
        <string> Date Added</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="image.qrc">:/img/img/up4.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="lineEdit_ExpireDate">
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 121, 87);
color: rgb(216, 216, 216);
</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string> Expire Date</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QPushButton" name="pushButton_UpdateAccount">
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft Sans Serif</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255,0);
color: rgb(0, 63, 191);</string>
       </property>
       <property name="text">
        <string>Update Account</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/img/img/1985763-200.png</normaloff>:/img/img/1985763-200.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_AccountType">
         <property name="font">
          <font>
           <family>Microsoft Sans Serif</family>
           <pointsize>7</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 63, 191);</string>
         </property>
         <property name="text">
          <string>AccountType</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_Admin">
         <property name="font">
          <font>
           <family>Microsoft Sans Serif</family>
           <pointsize>7</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 63, 191);</string>
         </property>
         <property name="text">
          <string>ADMIN</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_Member">
         <property name="font">
          <font>
           <family>Microsoft Sans Serif</family>
           <pointsize>7</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 63, 191);</string>
         </property>
         <property name="text">
          <string>MEMBER</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="8" column="2">
      <widget class="QPushButton" name="pushButton_search">
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft Sans Serif</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255,0);
color: rgb(0, 63, 191);</string>
       </property>
       <property name="text">
        <string>search</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/img/img/search.png</normaloff>:/img/img/search.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_Name">
       <property name="font">
        <font>
         <family>Microsoft Sans Serif</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 67, 99);</string>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLineEdit" name="lineEdit_DateAdded">
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(106, 101, 255);
color: rgb(216, 216, 216);</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string> Date Added</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit_MemberId">
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 121, 87);
color: rgb(216, 216, 216);</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string> Member ID</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_Id">
       <property name="font">
        <font>
         <family>Microsoft Sans Serif</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 67, 99);</string>
       </property>
       <property name="text">
        <string>Member ID</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
